### PR TITLE
added compare plugin

### PR DESCRIPTION
**Pull request by chcg**

### CODE REVIEW

File: plugins/plugins64.xml [hunk 1]
```diff
@@ -61,4 +61,25 @@
...
+	<plugin name="Compare">
+		<x64Version>2.0.0</x64Version>
+		<description>A very useful compare plugin to show the differences between 2 files (side by side).</description>
+		<author>Ty Landercasper, Jean-Sebastien Leroy, Pavel Nedev</author>
+		<sourceUrl>https://github.com/jsleroy/compare-plugin</sourceUrl>
+		<latestUpdate>Remove the dependance of other dll</latestUpdate>
+		<install>
+			<x64>
+				<download>https://github.com/pnedev/compare-plugin/releases/download/v2.0.0/ComparePlugin_v2.0.0_x64.zip</download>
+				<copy from="ComparePlugin.dll" to="$PLUGINDIR$\" validate="true" recursive="true"/>
```
Comment (bruderstein) [inline plugins/plugins64.xml:73]: You won't need `recursive` on this - it's just a single file

File: plugins/plugins64.xml [hunk 2]
```diff
@@ -61,4 +61,25 @@
...
+				<copy from="ComparePlugin\*.*" to="$PLUGINDIR$\ComparePlugin\" validate="true" recursive="true"/>
+			</x64>
+		</install>
+		<remove>
+			<x64>
+				<delete file="$PLUGINDIR$\ComparePlugin.dll"/>
```
Comment (bruderstein) [inline plugins/plugins64.xml:79]: No need to remove the plugin DLL - that's done automatically

Comment (chcg) [reply plugins/plugins64.xml:79]: This is just a takeover of the x86 unicode config, so already there these issues arise, should I correct them in the admin system? I have contacted the author at https://github.com/pnedev/compare-plugin/issues/127.

Comment (bruderstein) [reply plugins/plugins64.xml:79]: ok no worries. For the easy things like this it's probably easiest just to correct it in the admin system.

Comment (pnedev) [reply plugins/plugins64.xml:79]: Hi guys,

Thank you.
I'll update the config in the x86 list according to Dave's comment.

BR

**Pull request merged**